client/vscode/src/identityActivation.ts

```diff
@@ -0,0 +1,69 @@
+import { commands, window } from 'vscode'
+import {
+    ListProfilesError,
+    ListProfilesParams,
+    ListProfilesResult,
+    ProfileKind,
+    SsoTokenChangedParams,
+    UpdateProfileError,
+    UpdateProfileParams,
+    UpdateProfileResult,
+} from '@aws/language-server-runtimes/protocol'
+
+import { LanguageClient, ProtocolNotificationType, ProtocolRequestType } from 'vscode-languageclient/node'
+
+const ssoTokenChangedRequestType = new ProtocolNotificationType<SsoTokenChangedParams, void>(
+    'aws/identity/ssoTokenChanged'
+)
+
+const listProfilesRequestType = new ProtocolRequestType<
```

Comment:
nit: I think it's not necessary to re-define request types here, since they are exported from runtimes/protocol https://github.com/aws/language-server-runtimes/blob/2231a9bee86eb5c91a909595a17e582dac915d48/runtimes/protocol/identity-management.ts#L106

Reply:
Actually, it is.  

When attempting to use the types defined in language-server-runtimes an "Unknown parameter structure by name" error.  It's discussed in the links below.  It would seem that there are two different versions of vscode-jsonrpc being imported.  I suspect the one version from @aws/language-server-runtimes and the other from vscode-languageclient/node.  

Looking at the other activation code in that folder, some recreate the types (e.g. credentialsActivation.ts, futureTypes.ts), some add ".method" to the end of the type to call a different overload of sendRequest that doesn't raise the same error (e.g. customizationActivation.ts), and one, chatActivation.ts, uses the type directly imported from language-server-runtimes with the import of LanguageClient as well.  

I wasn't sure how to test chatActivation.ts to see if actually works or results in the same error.  When I removed the ".method" in customziationActivation.ts, the same error occurred, so it's not specific to the identityActivation.ts file.

For now, I'll just add ".method" to the end of the types and update in the next PR.

https://github.com/microsoft/vscode-docker/issues/2601
https://github.com/TypeFox/monaco-languageclient/issues/254
https://github.com/TypeFox/monaco-languageclient/issues/269

Reply:
Got it, thanks for explanation. This is something we could improve later to simplify this integration with VSCode